### PR TITLE
Bump the rev of protobuf supported to 4.21.8

### PR DESCRIPTION
**Pull request by nipunn1313**
Bumping rev of mypy-protobuf incoming next

This change is fairly minor - just some comment changes in the generated stubs.

### CODE REVIEW

**Comment by AlexWaygood** [comment]
We were wondering about removing `google/__init__.pyi` in our next major version bump for our protobuf stubs. Would you be interested in making that change as part of this PR? And if so -- would you be okay if we held off merging this until mypy 0.990 is released (should be in the next few weeks)? `--namespace-packages` will be the default in mypy 0.990, so it will make removing `google/__init__.pyi` much less painful for users (but we'd still rather make the change along with a major version bump, just to be sure).

**Comment by nipunn1313** [comment]
Sure that works.

One idea could be to keep this at `3.21.8` until then - just to unblock/decouple this work from the `__init__` work - since this piece is not actually a breaking change.

File: stubs/protobuf/METADATA.toml [hunk 1]
```diff
@@ -1,2 +1,2 @@
-version = "3.20.*"
+version = "4.21.*"
```
Comment (nipunn1313) [inline stubs/protobuf/METADATA.toml:1]:
```suggestion
version = "3.21.*"
```

Sort of a holdover in the meantime

Comment (nipunn1313) [reply stubs/protobuf/METADATA.toml:1]: (if you like the idea, feel free to commit it and merge - otherwise we can just wait.)

Comment (AlexWaygood) [reply stubs/protobuf/METADATA.toml:1]: Sure, let's do it!

File: stubs/protobuf/METADATA.toml [hunk 2]
```diff
@@ -1,2 +1,2 @@
-version = "3.20.*"
+version = "3.21.*"
```
Comment (AlexWaygood) [inline stubs/protobuf/METADATA.toml:1]:
```suggestion
version = "3.20.*"
```

**Pull request merged**